app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class DownvotesController < ApplicationController
+  before_action :find_status, only: [:create, :destroy]
+  def create
+    unless current_user.disliked? @micro_post
+      @micro_post.disliked_by current_user
+      respond_to do |format|
+        format.html { redirect_to user_path(current_user), success: t(".dislike") and return }
```

Comment:
and return để làm j?

Reply:
để tránh bị lỗi khỉ gió j đó

Reply:
là lỗi j

Reply:
1 điều kì lạ là từ khi e tách ra 2 controller cho votes thì e vừa check k còn bị lỗi đó nữa :3